external/mdal/3rdparty/libplyxx/libplyxx.cpp

```diff
@@ -51,8 +51,8 @@ SOFTWARE.
 #include <string>
```

Comment:
revert changes to this file and make it as PR in MDAL. probably changes made in QGIS were not ported to MDAL master...